coupon_commercial_partner_applicability/tests/test_sale_coupon_apply_commercial_partner.py

```diff
@@ -42,10 +42,19 @@ def setUpClass(cls):
 class TestSaleCouponApplyCommercialPartner(SaleCouponApplyCommercialPartnerCase):
     def test_sale_coupon_apply_commercial_partner(self):
         self.assertEqual(self.coupon.state, "new")
-        msg = self.coupon._check_coupon_code(self.sale)
+        order = self.sale
```

Comment:
This module doesn't depend on sale now, so you can't make a test with that model

Reply:
thanks :)